typescript/infra/scripts/helloworld/kathy.ts

```diff
@@ -57,6 +89,32 @@ async function sendMessage(
   console.log(JSON.stringify(receipt.events || receipt.logs));
 }
 
+function getPushGateway(): Pushgateway | null {
+  const gatewayAddr = process.env['PROMETHEUS_PUSH_GATEWAY'];
+  if (gatewayAddr) {
+    return new Pushgateway(gatewayAddr, [], metricsRegister);
+  } else {
+    console.warn(
+      'Prometheus push gateway address was not defined; not publishing metrics.',
+    );
+    return null;
+  }
+}
+
+async function submitMetrics() {
+  const gateway = getPushGateway();
+  if (!gateway) return;
+
+  const { resp, body } = await gateway.push({ jobName: 'relayer_funder' });
+  const statusCode =
+    typeof resp == 'object' && resp != null && 'statusCode' in resp
+      ? (resp as any).statusCode
+      : 'unknown';
```

Comment:
```suggestion
  const statusCode = resp?.statusCode ?? 'unknown';
```

Reply:
the ternary is very hard to read
are there no types for push https://github.com/prometheus/pushgateway?